jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java

```diff
@@ -341,6 +344,15 @@ private String dump(File specificOutDir, String fileNameFormatString, String con
         }
     }
 
+    static boolean svgFlamegraphs(String ver) {
+        if (ver.startsWith("1")) {
```

Comment:
well. they dropped svg @ version 1.8.5 so check is not valid :/

Reply:
See the discussion here: jvm-profiling-tools/async-profiler#427 -- I think Andrei made a mistake with tagging the repo, but actual 1.8.5 release should be fine.

Reply:
lets change it to .startsWith("1.") :)

Reply:
Right.